app/controllers/insured/fdsh_ridp_verifications_controller.rb

```diff
@@ -8,6 +8,7 @@ class FdshRidpVerificationsController < ApplicationController
     before_action :set_consumer_bookmark_url, only: [:service_unavailable, :failed_validation]
 
```

Comment:
I could also see using a before_action for `:new`, `:wait_for_primary_response`, `: wait_for_secondary_response`, `:primary_response`, `:secondary_response`, `:create`, `check_primary_response_received `, `service_unavailable `, and `failed_validation` to DRY up the actions using the `complete_ridp?` auth method. 

I think I've seen that pattern in a few other controllers. Curious what your take is on that: any reason to avoid doing that here, or in general?

Other than that question, LGTM!

Reply:
These before actions are for legacy controllers that do weird stuff - they manipulate something with URLs and the session.  My goal right now is simple: leave them alone unless I *must* change them.

Reply:
Fair enough, LGTM